modules/aerodyn/src/AeroDyn_IO.f90

```diff
@@ -1541,6 +1557,12 @@ SUBROUTINE AD_PrintSum( InputFileData, p, p_AD, u, y, ErrStat, ErrMsg )
       WRITE (UnSu,OutPFmt)  I, p%BldNd_OutParam(I)%Name, p%BldNd_OutParam(I)%Units
    END DO
 
+   ! Buoyancy parameters
+   WRITE (UnSu,'(//,A,/)')  'Buoyancy parameters:'
+   WRITE (UnSu,FmtDat ) '    Hub volume                     (m^3)', p%VolHub ! Buoyancy volume of the hub
+   WRITE (UnSu,FmtDat ) '    Nacelle volume                 (m^3)', p%VolNac ! Buoyancy volume of the nacelle
+   WRITE (UnSu,FmtDat ) '    Total blade volume, calculated (m^3)', p%VolBl  ! Buoyancy volume of all blades
+   WRITE (UnSu,FmtDat ) '    Tower volume, calculated       (m^3)', p%VolTwr ! Buoyancy volume of the tower
```

Comment:
I know the rest of the summary file is not in yaml, but we could progressively introduce it with  `yaml_write_var` from nwtc-library/yaml.f90.

Reply:
Sounds good, I changed the format to use yaml.